test/verify/check-metrics

```diff
@@ -1026,6 +1026,52 @@ BEGIN {{
 
         m.execute("rm /tmp/hogged")
 
+        # Test Podman containers
+        container_name = "pod-mem-hog"
+        m.execute(f"podman run --rm -d --name {container_name} quay.io/libpod/busybox /bin/sh -c 'cat /dev/zero | head -c 300m | tail | sleep infinity'")
```

Comment:
useless use of cat :cat2: 

So that is the POSIX shell way of allocating 300 MB of RAM? l33t! Doesn't busybox have `dd`?

Reply:
It has dd, what did you envision?

Reply:
As `dd` is used for the cpu hog test so hmm? I stole the solution from [SO](https://unix.stackexchange.com/a/254976) and adjusted it to work with busybox sh. The `tail` is required to keep the data into memory which is quite interesting :)

Reply:
I meant `dd if=/dev/zero bs=1m count=300` to read 300 MB of zeros. But indeed `head -c 300m < /dev/zero` is just as good, I wasn't aware that `head` can deal with prefixes as well. TIL!

(Nice trick with tail|sleep)